internal/commands/accesslist/update.go

```diff
@@ -0,0 +1,151 @@
+package accesslist
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type updateInputs struct {
+	cli.ProjectInputs
+	Address    string
+	NewAddress string
+	Comment    string
+}
+
+// CommandMetaUpdate is the command meta for the `accesslist update` command
+var CommandMetaUpdate = cli.CommandMeta{
+	Use:         "update",
+	Display:     "accesslist update",
+	Description: "Modify an IP address or CIDR block in the Access List of your Realm app",
+	HelpText: `Changes an existing entry from the Access List of your Realm app. You will be
+prompted to select an IP address or CIDR block if none is provided in the initial command.`,
```

Comment:
maybe this last sentence should read as: "You will be prompted to select an IP address or CIDR block to update if neither is specified."

also, this second line is over the 80 char limit... depending on how you potentially reword it here, I'd go in and double check the line widths after that